imap_upstream.adoc

```diff
@@ -16,13 +16,11 @@ IMAP related configuration options are available:
 
 The available balancing algorithms are as follows:
 
-* ClientIpHash
+* AccountIdHash
 
-  Selects an IMAP server server based on a hash of the client IP address.
-  The hashing algorithm used is a simple modulus of the client's ip address as an integer with the number of
-   configured IMAP servers. This will always return the same 'index' for the same client ip address.
-
-   Note: Extra configuration is necessary if a NAT proxy is being utilized in the environment.
+  Selects an IMAP server server based on a hash of the account id of the user making the request.
+  The hashing algorithm used is a simple modulus of the account id as an integer with the number of
+   configured IMAP servers for the mailbox hosting the account. This will always return the same 'index' for the same account id.
```

Comment:
nit: line lengths are a bit all over the place (not just here, but in our adocs in general), doesn't matter much, but I do find it somewhat annoying ... if you ever go look at something like linux kernel docs it seems like they try to at least shoot for the 80 char line length limit, I think something like that would be nice for our docs too.  Think emacs ```fill-paragraph``` or vim ```gq``` http://vim.wikia.com/wiki/Automatic_formatting_of_paragraphs